home.md

```diff
@@ -4,8 +4,8 @@ title: Home
 permalink: home/
 ---
 
-## What is Mechtronica?
+### What is Mechtronica?
 Mechtronica is a table top RPG set in a post-apocalyptic sci-fi universe focusing heavily on mechs, large bipedal piloted robots. At the heart of Mechtronica is mech building, most of the rules center around how you build a functional mech and we strive to make it feel both highly customizable and yet constrained in pseudo-realistic ways.
 
-## Status
-Mechtronica is currently in development and this website is not yet complete. We hope to have this finished very soon so the system can be publicly available for testing and feedback.
\ No newline at end of file
+### Status
+Mechtronica is currently in "Public Beta". The rules are complete and everything you need to play is available on this website. However some numbers may still need fine tuning, the lore is a little lacking in content, and some minor rules may change. We would love any feedback you may have, you can email us at [feedback@mechtronica.net](mailto:feedback@mechtronica.net) or submit an issue on our [GitHub repository](https://github.com/mechtronica/mechtronica.net/issues).
```

Comment:
- Maybe make Public Beta bold instead of using quotes? Your call on that.
- Fix comma splice in the last sentence.

Reply:
Done